nucypher/policy/revocation.py

```diff
@@ -27,26 +27,20 @@
 from nucypher.crypto.umbral_adapter import Signature, PublicKey
 from nucypher.policy.kits import MessageKit
 from nucypher.policy.maps import AuthorizedKeyFrag
+from nucypher.utilities.versioning import Versioned
 
 
-class Revocation:
+class RevocationOrder(Versioned):
     """
     Represents a string used by characters to perform a revocation on a specific
     Ursula. It's a bytestring made of the following format:
     REVOKE-<arrangement id to revoke><signature of the previous string>
     This is sent as a payload in a DELETE method to the /KFrag/ endpoint.
     """
 
-    PREFIX = b'REVOKE-'
-    revocation_splitter = BytestringSplitter(
-        (bytes, len(PREFIX)),
-        checksum_address_splitter, # ursula canonical address
-        (bytes, AuthorizedKeyFrag.ENCRYPTED_SIZE),  # encrypted kfrag payload (includes writ)
-        signature_splitter
-    )
-
     def __init__(self,
-                 ursula_checksum_address: ChecksumAddress,  # TODO: Use staker address instead (what if the staker rebonds)?
+                 ursula_checksum_address: ChecksumAddress,
+                 # TODO: Use staker address instead (what if the staker rebonds)?
```

Comment:
Not really the topic of this PR, but I think it is the staker address - it comes from the treasure map